### ISSUE TITLE
Not finishing execution on PowerShell

### ISSUE THREAD

**Issue by corujoraphael** [NatronGitHub/Natron#633]
Hi team,
I m not sure it s an issue, i probably might do something wrong, but here is my question.
When I run "Natron -b .\LowerThird_01.ntp" in PowerShell everything works fine, but the script doesn't finish the execution. It gets blocked on the last message waiting for some action. The last lines of process feedback are:

```
Write1 ==> Frame: 297, Progress: 99.0%, 1.9 Fps, Time Remaining: 1 seconds
[libx264 @ 000000000a1be9c0] non-strictly-monotonic PTS
Write1 ==> Frame: 298, Progress: 99.3%, 1.9 Fps, Time Remaining: 1 seconds
[libx264 @ 000000000a1be9c0] non-strictly-monotonic PTS
Write1 ==> Frame: 299, Progress: 99.7%, 1.9 Fps, Time Remaining: 0 second
[libx264 @ 000000000a1be9c0] non-strictly-monotonic PTS
Write1 ==> Frame: 300, Progress: 100.0%, 1.9 Fps, Time Remaining: 0 second
Write1 ==> Rendering finished
```

Do you have any flags you can put to stop waiting?

**Comment by rodlie** [comment]
Does the same happen in regular cmd.exe?

Windows still has some issues related to deadlock during project close, as far as I know this is mostly a third-party issue.

**Comment by devernay** [comment]
Third party meaning OpenImageIO? It would be nice to have a project to reproduce this.

**Comment by rodlie** [comment]
OIIO was a big problem, we couldn't even run the unit tests for more than a couple of minutes before a deadlock happened, this has now been fixed.

But I do notice sometimes when closing Natron that this happens, but it's very rare (on my setup/projects), I still believe it's related to OIIO.
